packages/lumx-react/src/components/post-block/PostBlock.tsx

```diff
@@ -14,27 +14,27 @@ import { GenericProps, getRootClassName, handleBasicClasses } from '@lumx/react/
  */
 interface PostBlockProps extends GenericProps {
     /* Actions elements to be transcluded into the component */
-    actions?: HTMLElement | ReactNode;
+    actions?: ReactNode;
     /* Atachments elements to be transcluded into the component */
-    attachments?: HTMLElement | ReactNode;
+    attachments?: ReactNode;
     /* Author element to be transcluded into the component */
-    author?: HTMLElement | ReactNode;
+    author?: ReactNode;
     /* Meta elements to be transcluded into the component */
-    meta?: HTMLElement | ReactNode;
+    meta?: ReactNode;
     /* Orientation. */
     orientation?: Orientation;
     /* Tags elements to be transcluded into the component */
-    tags?: HTMLElement | ReactNode;
+    tags?: ReactNode;
     /* Content text. Can be either a string, or sanitized html. */
```

Comment:
Some comments here still use `/*` instead of the JSDoc `/**` format

Reply:
Yes I saw that but I planned to fix the JSDoc in another PR to let this one more readable.